finch/processes/wps_xclim_indices.py

```diff
@@ -37,6 +36,7 @@ class XclimIndicatorBase(FinchProcess):
     """
 
     xci = None
+    allvars = []
```

Comment:
Hum, I'm not sure, but having a mutable default on the class, outside the  `__init__` seems weird.
```suggestion
```